website/docs/guides/best-practices/how-we-style/5-how-we-style-our-yaml.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How we style our YAML
+id: 5-how-we-style-our-yaml
+---
+
+## YAML Style Guide
+
+- 2️⃣ Indents should be two spaces
+- ➡️ List items should be indented
+- 🆕 Use a new line to separate list items that are dictionaries where appropriate
+- 📏 Lines of YAML should be no longer than 80 characters.
+- 🛠️ Use the [dbt JSON schema](https://github.com/dbt-labs/dbt-jsonschema) with any compatible IDE and a YAML formatter (we recommend [Prettier](https://github.com/dbt-labs/dbt-jsonschema)) to validate your YAML files and format them automatically.
```

Comment:
The prettier link is also to the JSON Schema GitHub page

Reply:
woopsie daisy ty